pyanaconda/modules/runtime/scripts/runtime.py

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from pyanaconda.core.configuration.anaconda import conf
+from pyanaconda.modules.common.errors.runtime import ScriptError
+from pyanaconda.modules.common.task import Task
+from pyanaconda.anaconda_loggers import get_module_logger
+
+from pykickstart.constants import KS_SCRIPT_POST
+
+log = get_module_logger(__name__)
+
+
+class RunScriptsTask(Task):
+    """Task for running scripts."""
+
+    def __init__(self, script_type, scripts):
+        """Create a new task.
+        :param script_type: type of scripts to be run
```

Comment:
Multiline doc strings should have a blank line after first summary line. Please fix this issue on the other places too.